src/components/Navigation/Navigation.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```

Comment:
tego importu nie trzeba

Reply:
Znowu ja miałem błąd jak nie miałem tego importa xD ale tylko był wężyk na czerwono to nie ma problemu. Poprawię,